packages/connectivity_plus/connectivity_plus/pubspec.yaml

```diff
@@ -32,7 +32,7 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  connectivity_plus_platform_interface: ^1.2.1
```

Comment:
because I published a release of 1.2.2 manually without using melos, as for some reason the current changes in main were never picked up by the melos tool